ft/ft.cc

```diff
@@ -435,7 +435,8 @@ int toku_read_ft_and_store_in_cachefile (FT_HANDLE ft_handle, CACHEFILE cf, LSN
     }
 
     int fd = toku_cachefile_get_fd(cf);
-    int r = toku_deserialize_ft_from(fd, max_acceptable_lsn, &ft);
+    const char *fn = toku_cachefile_fname_in_env(cf);
```

Comment:
"auto" only started creeping into FT just before we took ownership and there really isn't a defined policy other than Google code style guidelines which are somewhat ambivalent w.r.t. "auto". I'm not particularly thrilled with the idea of it in this use case as IMHO it decreases readability by hiding the nature of the true type and forces an observer to go look up the function signature if they need to understand what the type is and what they can do with it.